views/admin.functions.php

```diff
@@ -49,6 +49,24 @@ function sailthru_html_text_input_callback( $args ) {
 
 } // end sailthru_html_text_input_callback.
 
+function sailthru_html_hidden_text_input_callback( $args ) {
+
+	$collection    = $args[0];
+	$option_name   = $args[1];
+	$default_value = $args[2];
+	$html_id       = $args[3];
+
+	$options = get_option( $collection );
+
+	// Make sure the element is defined in the options. If not, we'll use the preferred default.
+    $value = $default_value;
+    if ( isset( $options[ $option_name ] ) ) {
+        $value = $options[ $option_name ];
+    }
+	echo '<input type="password" id="' . esc_attr( $html_id ) . '" name="' . esc_attr( $collection ) . '[' . esc_attr( $option_name ) . ']" value="' . esc_attr( $value ) . '" class="regular-text" />';
```

Comment:
Looks like the whitespace (tabs/spaces) aren't right for lines 62-65. Can you please fix this?